### ISSUE TITLE
Importing Marketdata From An Existing RN Generated CSV

### ISSUE THREAD

**Issue by SPCStarFighter** [Duke-Jones/ED-IBE#34]
Hiya,
I am (finally) able to get back to Elite:Dangerous & am quite pleased (so far) with the way things have progressed.
I had an issue in that I was used to using RN for the market data and having come back to find that it (effectively) doesn't exist anymore was quite a shock.
It was suggested on the ED Forums that I try out your ED:IBE & I have been looking into that.
I noticed on Tuesday 16th May that I could import my old RN generated CSV data (there are 100,474 lines in the file) & so I set that import going ......
THE PROBLEM .........
It's still running ......................
As you can see it's now Thursday morning which means the import has been running for 48 hours so far.....
Any idea when it'll finish ?
It's still updating the screen with messages such as :-
analysing data ........ : 100% (111 of 111)
converting ...... : 100% (111 of 111)
updating ...... : 100% (111 of 111)
delete prices of no longer existing commodities on station - deleted prices = 0

hence I assume it IS still importing the CSV data, but really ????? How long should it take to import less than 101,000 lines of "standard" RN generated CSv data ? I mean there isn't that much in a line :- 

10 KAPPA PEGASI;Morin Dock;Advanced Catalysers;3340;;1361193;;;;2015-02-28T15:58;<From EDDN>;

the above is just one typical line from my CSV so you can see it is the "standard" stuff & nothing that your ED:IBE would have issues with.

Is there anyway of "interpreting" the "analysing/converting/updating" information to figure out how far it's got through the CSV ?

I notice that the "tbcommoditydata.idb" in the ED:IBE\database\data directory is growing & is currently up to 54,344KB

Any help would be gratefully received as I (obviously) don't really want to "cancel the operation".
Incase it helps, my computer is an i7 7700K overclocked slightly to run at 4.5GHz with 32GB (2x16GB) 3200MHz RAM on a Gigabyte Z270 Gaming 7 motherboard with a Gigabyte GTX 1080 graphics card - so, as you can see - it's not as if the PC is "slow".

Look forward to hearing back from you or else e-mail me directly at :- 
Steve@NetData-Systems.co.uk
Thanks.

**Comment by Duke-Jones** [comment]
Hi
Sorry for my late response. Anything new about it ?

**Comment by SPCStarFighter** [comment]
No. In the end I did give up & cancel the operation just so I could play ED. I've decided that I'm going to uninstall ED:IBE remove all traces of it, then reinstall it, import my data first THEN do the import of ALL DATA AVAILABLE from the net & see if that's quicker.
Will post the results here once I've done it. (Might be a while though as I have a few other work jobs to do urgently.)
